deploy/plays/tasks/dockerBuild.yml

```diff
@@ -0,0 +1,18 @@
+---
+  - name: Build main server docker image
+    become: yes
+    docker_image:
+      name: "main_server"
+      path: "../../main_server"
```

Comment:
for the sake of consistency, it is better to go with absolute paths.

Reply:
I have updated this to the playbook directory path now.